static/style.css

```diff
@@ -450,7 +463,7 @@ input.inline-content-editable:focus {
   }
 
 =======
-  
+
 >>>>>>> ea6a1c13745b2c341d43836a122f63fc9f048f18
```

Comment:
It looks like some merge conflict stuff got into the style.css file ... I'm not sure if that causes major problems or not? If you do a search for ">>>" or "<<<" you should be able to find any stray markers and nuke em.

Reply:
Oh those were totally my fault lol, I found the commit that introduced them. I'll fix em tomorrow!